spec/log_parser_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../components/log_parser'
+
+describe '#open_file' do
+  it 'check first line' do
+    file_path = "src/games.log"
+
+    file = LogParser.new(file_path)
+    first_log_line = file.open_file[0]
+    first_expected_line = "  0:00 ------------------------------------------------------------"
+    expect(first_log_line).to eq(first_expected_line)
```

Comment:
você deve testar a chamada do método `first_line` vinda do objeto de parser, algo como:
```suggestion
    expect(LogParser.new(fixture_file_path).first_line).to eq(first_expected_line)
```